### PR TITLE
[ENG-382] Refactor python wrappers for coming enterprise features

### PR DESCRIPTION
**Pull request by arthurp**
**Breaking change**: All python programs now need to call `katana.local.initialize()` to setup the Katana runtime.

I chose `katana.local` a the package for the "shared memory" runtime. I'm open to other names, but I couldn't think of anything better and it aligns well with `katana.distributed`.

This PR does not include testing or error checking for what happens if a user tries to use katana without calling `initialize`. That will be needed. If waiting is acceptable I will create a tracking github issue. There is no clear place to put a TODO in the code since the issue affects every entry point to the katana Python API.

### CODE REVIEW

File: python/katana/_property_graph.pyx [hunk 1]
```diff
@@ -336,32 +274,83 @@ cdef class PropertyGraph:
...
+    @staticmethod
+    cdef PropertyGraph make(shared_ptr[_PropertyGraph] u):
+        f = <PropertyGraph>PropertyGraph.__new__(PropertyGraph)
+        f._underlying_property_graph = u
+        return f
+
+    @staticmethod
+    def from_graphml(path, uint64_t chunk_size=25000):
+        """
+        Load a GraphML file into katana form
```
Comment (ddn0) [inline python/katana/_property_graph.pyx:342]: qq: we still support GraphML?

**Review by ddn0 — APPROVED** [review]
seems reasonable to me

File: python/katana/_property_graph.pyx (hunk 1 above)
Comment (arthurp) [reply python/katana/_property_graph.pyx:342]: I have no idea. I was asked to add this by @bowu and there was code implementing it in the graph-convert tools.

Comment (bowu) [reply python/katana/_property_graph.pyx:342]: It's an ask from Intuit.

**Pull request merged**